raspberrypi3-64.coffee

```diff
@@ -8,7 +8,6 @@ module.exports =
 	name: 'Raspberry Pi 3 64bits'
 	arch: 'aarch64'
 	state: 'experimental'
-	isDefault: true
```

Comment:
@LucianBuzzo can you detail a little bit why is this change needed? Preferably in the commit log?